app/controllers/api/v2/people_controller.rb

```diff
@@ -35,6 +52,17 @@ def validate_include_params
           end
         end
       end
+
+      def person_params
+        params.require(:data).permit(PERMITTED_PERSON_PARAMS).to_h
+      end
+
+      def new_person_attributes
+        person_params[:attributes].merge(
+          ethnicity_id: params.require(:data).dig(:relationships, :ethnicity, :data, :id),
```

Comment:
I'm wondering about this, the pattern in other controllers has been to find the records for validation reasons, maybe we can do the same here?

Reply:
good point! I'll add that!